openstack/cinder/values.yaml

```diff
@@ -32,8 +32,8 @@ pod:
   resources:
     api:
       requests:
-        cpu: "1500m"
-        memory: "5Gi"
+        cpu: "1000m"
```

Comment:
Are you sure it will use less than 1,5 CPUs now? I'd think it uses the same, just less memory.

Reply:
hmm, I'd actually expect it to use less cpu will fewer workers.
But those are not the limits, rather the requests for scheduler for a better placement, so it will not cap even if real usage goes to 2+ cpu.

Reply:
yeah, still. I expect it didn't use all the workers before and will not use now. So the usage should stay stable.
If anything, the number might be reduced with a bigger number of pods, because requests are spread to more pods.

Reply:
Ok, back to 1500